pom.xml

```diff
@@ -49,7 +49,9 @@
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
 
-        <asciidoctorj.version>1.5.3.2</asciidoctorj.version>
+        <asciidoctorj.version>1.6.0-alpha.3</asciidoctorj.version>
```

Comment:
Mmmmh OK. Looks like they messed up the PDF plugin. And they moved the 1.5 branch of the plugin to the 1.6 branch of asciidoctor-pdf :/. So we don't really have a handy solution here.

To be honest, I'm a bit skeptic of going the ant uberjar route. We already have some dependency management in our build.xml. So what I'm wondering is if we could build a asciidoctor-ant-runner that would only contain the asciidoctor-ant classes and wouldn't include the dependency. Thus we could provide them as a classpath in our ant file and this would simplify the upgrade of one of the components. I created an issue upstream to see what they think about that.

After discussing it with Gunnar, the idea would be to copy the ant task of the asciidoctor-ant project in the beanvalidation-spec project. No need for a Maven project, just put the necessary files in a ant/ directory. You can see how we used to compile java files here: https://github.com/beanvalidation/beanvalidation-spec/commit/346fdf7c10f970fb20d49e8793170b92612a129e#diff-2cccd7bf48b7a9cc113ff564acd802a8
so it should be easy to mimic it to compile the ant task. There is only one file to compile (and the antlib.xml have to also be available).

Then we would run our ant task with the necessary dependencies in the classpath. You already have a few examples on how to download a Maven dependency.

This way, we will be able to use the right dependencies and be consistent with what we do on HV with Maven.

Does it look like a good plan to you?